lib/src/scanner.dart

```diff
@@ -270,7 +273,15 @@ class NgScanner {
       _state = NgScannerState.scanInterpolation;
       return new NgToken.interpolationStart(_current.offset);
     }
-    return handleError();
+
+    var override = _current;
+    if (_current.type == NgSimpleTokenType.text &&
+        _reader.peekType() == NgSimpleTokenType.mustacheEnd) {
+      override = _reader.peek();
+    }
+    var overrideMessage = 'Unopened mustache';
```

Comment:
Could use a comment on why this error case occurs. For instance, when would _current.type be text?